pom.xml

```diff
@@ -76,7 +77,7 @@
         <!-- Properties for the frontend-maven-plugin -->
         <node.version>v8.0.0</node.version>
         <yarn.version>v1.2.1</yarn.version>
-        <netty.version>4.1.5.Final</netty.version>
+        <netty.version>4.1.16.Final</netty.version>
```

Comment:
❓ curious, why the netty version change?

Reply:
To match DDF and get us to closer to a recent version.

Reply:
Ok, that is probably fine.  We do use netty in the video app but I'd be surprised if a minor version bump would be a problem